### PR TITLE
Us122 lm

### PR DESCRIPTION
**Pull request by dlehrm**

### CODE REVIEW

File: src/test/java/com/crm_app/step_definitions/US122_LMStepDef.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    @When("user clicks the Activity Stream tab")
+    public void user_clicks_the_activity_stream_tab() {
+        us122LmPage.activityStream.getText();
+        Assert.assertTrue("Activity Stream tab is not displayed Activity Stream page", us122LmPage.activityStream.isDisplayed());
+
+    }
+    @Then("user should see the MESSAGE tab on the Activity Stream Page")
+    public void user_should_see_the_message_tab_on_the_activity_stream_page() {
+        us122LmPage.activityStream.getText();
+        Assert.assertTrue("Message tab is not displayed on the Activity Stream page", us122LmPage.messageTab.isDisplayed());
```
Comment (Suelapepaj) [inline src/test/java/com/crm_app/step_definitions/US122_LMStepDef.java:27]: you have this assert here as well as in the step definition before, I think you should have in one place

File: src/test/resources/features/US122_LM.feature [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Feature: CRM Employee Page
+  Agile Story: User should be able to see all options on the Activity Stream Page
+
```
Comment (Suelapepaj) [inline src/test/resources/features/US122_LM.feature:3]: I would create a background here for these steps then you dont have to repeat yourself
Given user is on the CRM login page
When user enters Username and Password to login

**Comment by dlehrm** [comment]
Yes, that's a good idea, thank you.  I will rework it for more practice.
________________________________

**Comment by dlehrm** [comment]
Yes good point, thank you.
________________________________

File: src/test/resources/features/US122_LM.feature [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  @B30G21-123
+  Scenario: View the following options on the Activity Stream Page
+    Given user is on the CRM login page
+    When user enters Username and Password to login
+    And user clicks the Activity Stream tab
```
Comment (Suelapepaj) [inline src/test/resources/features/US122_LM.feature:8]: this is fine how you located these tabs for activity stream all separately but you could also locate them as a list of webelements and then have them all in one step definition, let me know know if that makes sense
    And user clicks the Activity Stream tab
      | Message        |
      | Task            |
      | Event         |
      | Poll        |
  | More         |

**Review by Suelapepaj — COMMENTED** [review]
great job

**Review by oabdolah — APPROVED** [review]

**Comment by oabdolah** [comment]
good job Lisa

**Pull request merged**